### DISCUSSION TITLE
Copy NPC

### DISCUSSION THREAD

**Discussion by Odoamar** [fmasa/wfrp-master#92] [Ideas]
I created one template NPC for the encounter but unfortunately, I must update all skills and items individually.  In my example: I decided to create a group of 5 Thugs (3 with melee weapons, 1 archer, 1 leader) - they should share almost all stats and skills except 1-2 archetype specific. 
In my opinion, simple copy NPC feature will be very useful in similar cases

**Comment by fmasa** [comment]
@Odoamar Great idea! This should be fairly easy as well.

**Reply by fmasa** [reply]
Closing, as this is now implemented (see issue https://github.com/fmasa/wfrp-master/issues/93)
